### DISCUSSION TITLE
NPCs wont shoot at riot shields

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13611] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Hostile NPCs won't shoot at riot shields

I guess nobody reported this because 2-handed riot shields arent popular 😉 

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/6b7ce208-45f2-4fe6-b75e-b8d6e8d36de9)

### Reproduction steps

1. Start Campaign
2. set reputation to -100 
3. hold a riot shield in front of enemy security

### Bug prevalence

Just once

### Single player or multiplayer?

multiplayer

### Version

Unstable

**Comment by Regalis11** [comment]
I couldn't find a way to reproduce this. :/

https://github.com/FakeFishGames/Barotrauma/assets/14351070/a9c1b4d3-c3d9-42dd-af70-c092ad2148ee

**Comment by LePiromano** [comment]
they open fire just fine for me too

**Comment by ubertpendragon** [comment]
The entire unstable campaign used and abused it but not a single one will comment on the ticket

**Comment by Regalis11** [comment]
I don't doubt there's some kind of a bug that's causing this, but it seems like it requires some specific circumstances to occur (maybe some specific server setting or something), and I unfortunately couldn't figure out what that is. :/

And to be clear, did this occur in single player or multiplayer, or both? The ticket says single player (and that's what I tested myself), but you mentioned it happening in a multiplayer session? Unstable or the stable version? The ticket says stable, but you mentioned an unstable campaign?

**Comment by Regalis11** [comment]
Hmm, I see that you're in the docking module in the screenshot? I believe that might be the explanation, security doesn't fire unless you try to get further into the outpost, regardless if you're holding a riot shield or not.

**Comment by ubertpendragon** [comment]
Sorry it was during the multiplayer unstable campaign, I've seen the riot shield used extensively to push and block hostile NPCs at <-90 reputation on hostile outposts, I did not personally use it

**Comment by Wheatley1001** [comment]
I can confirm that security do not engage with users who hold a riot shield in front of them. They will allow themselves to be pushed around until they, seemingly without reason nor warning, will decide to start firing at you. Me and my crew utilised this exploit to navigate hostile outposts that had questline specific missions involved (Jailbreak and the Tormsdale Report). Otherwise such missions are not possible to complete due to the immediately aggressive nature of the AI, which is an issue within itself.

**Comment by Regalis11** [comment]
Even though we weren't able to find a way to repro this, since we haven't heard any reports of this since, I'd suspect it's been fixed by the changes done to address https://github.com/FakeFishGames/Barotrauma/discussions/14871
